apps/src/templates/sectionAssessments/FreeResponseBySurveyQuestionContainer.jsx

```diff
@@ -0,0 +1,40 @@
+import React, {Component, PropTypes} from 'react';
+import FreeResponsesSurveyTable from './FreeResponsesSurveyTable';
+import {
+  getSurveyFreeResponseQuestions,
+} from './sectionAssessmentsRedux';
+import { connect } from 'react-redux';
+
+const freeResponseQuestionsPropType = PropTypes.shape({
+  questionText: PropTypes.string,
+  answers: PropTypes.array,
+});
+
+class FreeResponseBySurveyQuestionContainer extends Component {
+  static propTypes = {
+    freeResponsesByQuestion: PropTypes.arrayOf(freeResponseQuestionsPropType),
+  };
+
+  render() {
+    const {freeResponsesByQuestion} = this.props;
+    return (
+      <div>
+        <h2>Free response questions for this survey</h2>
```

Comment:
All of these headers are temporary until we create ones we actually want to keep.